extensions/wikia/Chat2/Chat.class.php

```diff
@@ -118,7 +118,8 @@ public static function banUser( $subjectUserName, User $adminUser, $time, $reaso
 				$action = self::BAN_CHANGE;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Use "static" keyword instead of "self". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS2037)